src/Dashboard/Withdraw/Balance.tsx

```
@@ -0,0 +1,100 @@
+    settings,
+    masterLoading,
+    withdrawRequests,
+}: {
+    bodyData: UseBalanceReturn;
+    settings: UseWithdrawSettingsReturn;
+    masterLoading: boolean;
+    withdrawRequests: UseWithdrawRequestsReturn;
+} ) {
+    if (
+        ! bodyData ||
+        ! bodyData.hasOwnProperty( 'isLoading' ) ||
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)